### ISSUE TITLE
The line of the timeline only appears until it reaches div's maximum height

### ISSUE THREAD

**Issue by yurifw** [RyanFitzgerald/vertical-timeline#8]
I have a div with the following css:
`#timeline{
	max-height: 400px;
	overflow-y:auto;
}`

when I turn it into a timeline, the following happens:
![image](https://user-images.githubusercontent.com/3476335/42578849-74e8d9ac-84fd-11e8-9ad2-c83e5614e2d7.png)

Seems normal until I scroll the div down:
![image](https://user-images.githubusercontent.com/3476335/42578919-99d6cc88-84fd-11e8-8a2c-bfc087a9afee.png)

**Comment by RyanFitzgerald** [comment]
Good catch, thank you! I'll get working on a fix unless you have already found one, then feel free to submit a PR.
